### PR TITLE
remove vendoring

### PR DESCRIPTION
**Pull request by mangas**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,8 +23,7 @@ endif
 
 LFLAGS ?= -X github.com/appvia/kev/pkg/${NAME}.Tag=${GIT_LAST_TAG} -X github.com/appvia/kev/pkg/${NAME}.GitSHA=${GIT_SHA} -X github.com/appvia/kev/pkg/${NAME}.Compiled=${BUILD_TIME} -X github.com/appvia/kev/pkg/${NAME}.Release=${VERSION} -X github.com/appvia/kev/pkg/${NAME}.GitBranch=${GIT_BRANCH}
 CLI_PLATFORMS=darwin linux windows
-CLI_ARCHITECTURES=amd64
-export GOFLAGS = -mod=vendor
+CLI_ARCHITECTURES=amd64 arm64
```
Comment (mangas) [inline Makefile:26]: adding this because of the m1 cpu

File: go.mod [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module github.com/appvia/kev
 
-go 1.14
+go 1.16
```
Comment (mangas) [inline go.mod:3]: is there a reason not to bump?

**Review by marcinc — COMMENTED** [review]
I think we need `vendor` dir excluded in .gitignore.

File: go.mod (hunk 2 above)
Comment (ezodude) [reply go.mod:3]: no

**Review by marcinc — APPROVED** [review]

**Pull request merged**